### PR TITLE
taxonomy: add missing polish translations

### PR DESCRIPTION
**Pull request by ArturLange**
### What

* Added some missing polish translations for ingredients (based on _unknown_ ingredients from the ingredients list and from personal experience)

### CODE REVIEW

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Comment by stephanegigandet** [comment]
@ArturLange : a translation cannot be associated to 2 different entries of a taxonomy, otherwise it won't compile:

Errors in the ingredients_processing taxonomy definition:
ERROR - pl:płatki already is a synonym of pl:płatki for entry en:flakes - pl:płatki cannot be mapped to entry en:chips / pl:chipsy
ERROR - pl:suszony already is a synonym of pl:w-proszku for entry en:powder - pl:suszony cannot be mapped to entry en:dried / pl:suszony
ERROR - pl:suszona already is a synonym of pl:w-proszku for entry en:powder - pl:suszona cannot be mapped to entry en:dried / pl:suszony
ERROR - pl:suszone already is a synonym of pl:w-proszku for entry en:powder - pl:suszone cannot be mapped to entry en:dried / pl:suszony

Can you try to find different translations for dried and powder, and for chips and flakes?

**Comment by ArturLange** [comment]
Sure, will fix that :)

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -63179,8 +63210,8 @@ protected_name_file_number:en: PGI-FR-0503
 protected_name_type:en: pgi
 origins:en: en:France
 wikidata:en:Q21427661
-# Le stockage des graines et la production de la pâte de moutarde doivent avoir lieu sur le territoire de la région Bourgogne, à savoir les quatre départements de la Côte-d'Or, de la Nièvre, de la Saône-et-Loire et de l'Yonne. 
-# La culture des graines doit s'effectuer dans une zone plus restreinte, au cœur de l'aire géographique précitée ; le Morvan et les monts du Mâconnais en sont exclus en raison de sols peu favorables à la culture3. 
+# Le stockage des graines et la production de la pâte de moutarde doivent avoir lieu sur le territoire de la région Bourgogne, à savoir les quatre départements de la Côte-d'Or, de la Nièvre, de la Saône-et-Loire et de l'Yonne.
+# La culture des graines doit s'effectuer dans une zone plus restreinte, au cœur de l'aire géographique précitée ; le Morvan et les monts du Mâconnais en sont exclus en raison de sols peu favorables à la culture3.
```
Comment (teolemon) [inline taxonomies/categories.txt:63214]:
```suggestion
# La culture des graines doit s'effectuer dans une zone plus restreinte, au cœur de l'aire géographique précitée ; le Morvan et les monts du Mâconnais en sont exclus en raison de sols peu favorables à la culture.
```

**Comment by stephanegigandet** [comment]
> Sure, will fix that :)

Thank you!

**Pull request merged**